semantic.cabal

```diff
@@ -304,41 +304,35 @@ library
                      , reducers ^>= 3.12.3
                      , semigroupoids ^>= 5.3.2
                      , servant ^>= 0.15
-                     , shelly
+                     , shelly >= 1.5 && <2
                      , split ^>= 0.2.3.3
                      , stm-chans ^>= 3.0.0.4
                      , template-haskell ^>= 2.14
                      , time ^>= 1.8.0.2
                      , unliftio-core
                      , unordered-containers ^>= 0.2.9.0
                      , vector ^>= 0.12.0.2
-                     , haskell-tree-sitter
```

Comment:
Did we drop `tree-sitter` altogether? That’s kind of cool.

Reply:
Nah, it was just duplicated.